common/djangoapps/course_modes/migrations/0005_auto_20151217_0958.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_modes', '0004_auto_20151113_1457'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
@bderusha This is removing the model field and adding a new one, but it won't actually end up touching the database. Django 1.8 migrations aren't super clear about what they're actually going to execute, so I tested locally to verify that nothing will be deleted.